api/v1alpha4/zz_generated.conversion.go

```diff
@@ -2170,18 +2187,16 @@ func Convert_v1alpha4_VPCSpec_To_v1beta1_VPCSpec(in *VPCSpec, out *v1beta1.VPCSp
 func autoConvert_v1beta1_VPCSpec_To_v1alpha4_VPCSpec(in *v1beta1.VPCSpec, out *VPCSpec, s conversion.Scope) error {
 	out.ID = in.ID
 	out.CidrBlock = in.CidrBlock
+	// WARNING: in.EnableIPv6 requires manual conversion: does not exist in peer-type
```

Comment:
@richardcase Halp. :D How do I make this work? :D I'm probably missing something from somewhere? :)

Reply:
Ah yes, conversions. Lets have a call tomorrow and i can take you through it.